test/models/ScreeningSpec.scala

```diff
@@ -0,0 +1,41 @@
+package models
+
+import org.scalatestplus.play.PlaySpec
+
+import java.time.LocalDateTime
+
+class ScreeningSpec extends PlaySpec {
```

Comment:
please propose a method for automated end-to-end test for you system

Reply:
I began writing test/controllers/TicketBookingSpec.scala (simulation of a user using a browser) - it is in the code - but I approached strange problems with setting a value for a dateTimeLocalField:  
"dateTimeLocalField("dateTime").value_= " - setting the value did not work
I tried to fix it multiple times but did not succeed. 
I would fix this class and provide some test-scenarios (proper and invalid). The only problem is with this dateTimeLocalField - I searched throughout the web and did read a lot about it and I think it should work - but it doesn't. Probably would have to try again.